superset/migrations/versions/2018-07-26_11-10_c82ee8a39623_add_implicit_tags.py

```diff
@@ -33,7 +33,7 @@
 from sqlalchemy import Column, DateTime, Enum, ForeignKey, Integer, String
 from sqlalchemy.ext.declarative import declarative_base, declared_attr
 
-from superset.models.tags import ObjectTypes, TagTypes
+from superset.tags.models import ObjectTypes, TagTypes
```

Comment:
> Looks good! One question, why the move of `/superset/models/tags.py` to `/superset/tags/models.py`?
> 
> Note: We are currently moving and deprecating all endpoints from `/supeset` to `/api/v1`

@dpgaspar I requested this move, as I feel we should move all models from the generic `/superset/models` dumping ground to the relevant context, in this case `/superset/tags`. The same applies to `utils` and other modules - IMO a util which is specific to a certain context, say `tags`, should be placed under `/superset/tags/utils.py` rather than `/superset/utils/tags.py`. A good example of this is `key_value` which follows this pattern. I got this inspiration from our frontend code organization SIP (see #13632), which has done wonders for the frontend codebase readability.

Reply:
cool, makes sense to me